history/subscriber/persister.py

```diff
@@ -268,6 +268,39 @@ def on_put(req, resp):
                 'Logging level must be DEBUG, INFO, WARNING, ERROR or CRITICAL!', 'level')
 
 
+def str2_bool(v):
+    """
+     this function basically only converts a string to boolean
```

Comment:
This function converts a string to bool if this is the case.
If the value received is of type bool, it is just returned. If it is a string and has one of these values "yes", "true", "t", "1" regardless of whether it is uppercase or not, it will return a bool with a true value.